doc/effects.md

```diff
@@ -224,6 +225,26 @@ tree after the specified delay has passed:
 final effect = RemoveEffect(delay: 10.0);
 ```
 
```

Comment:
Two new lines before a new section (not like that everywhere in our docs yet, but this file has it).

Reply:
Any particular reason for that? Just out of curiosity

Reply:
"My thinking is that we have 1 blank line between paragraphs, so we need to have 2 lines between sections in order to make the structure more visible. If not, it's hard to see section boundaries among the paragraphs.

This is similar to how in HTML we have larger margins around the <h2> / <h3> elements compared to the margins between the paragraphs." - Pasha